kimera_vio_ros/src/interfaces/stereo_interface.cpp

```diff
@@ -9,24 +9,23 @@ namespace interfaces
 const cv::Mat readRosImage(
   const sensor_msgs::msg::Image::ConstSharedPtr & img_msg)
 {
-  cv_bridge::CvImagePtr cv_ptr;
+  cv_bridge::CvImageConstPtr cv_constptr;
   try {
-    // TODO(Toni): here we should consider using toCvShare...
-    cv_ptr = cv_bridge::toCvCopy(img_msg);
+    cv_constptr = cv_bridge::toCvShare(img_msg);
```

Comment:
@ruffsl this commit msg caught my eye, bcs I think I tried toCvShare, but last time I tried there was something in the code modifying the images (probably in the frontend, which I am heavily refactoring now) , just FYI

Reply:
Ok, I'll revert that commit for the time being. Is there a ticket I could watch on that refactoring upstream?

Reply:
Ah, unfortunately, these tickets are in the private repo...

Reply:
@ToniRV , with the latest pushes, is this still an issue?

Reply:
I'm afraid it still is. I am working on this on another branch that hasn't been merged yet. Sorry for the delay!